content/chainguard/chainguard-images/authenticating.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Authenticating to Chainguard Registry"
+type: "article"
+description: "A guide on authenticating to the Chainguard Registry to get images"
+date: 2023-03-21T15:10:16+00:00
+lastmod: 2023-03-21T15:10:16+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 500
+toc: true
+---
+
+### Public Images
+
+All public Chainguard Images are available for free.
+
+However, logging in with a Chainguard account and authenticating when pulling from the registry provides a mechanism for Chainguard to contact you if there are any issues with images you are pulling.
+
+This may enable Chainguard to notify you of upcoming deprecations, changes in behavior, critical vulnerabilities and remediations for images you have recently pulled.
+
+### Authenticating with the credential helper
+
+The easiest way to configure authentication is to use the credential helper included with `chainctl`.
+
+First [install `chainctl`](../..//chainguard-enforce/how-to-install-chainctl/), and configure the credential helper:
```

Comment:
You can use the full path rather than relative here

```suggestion
First [install `chainctl`](/chainguard/chainguard-enforce/how-to-install-chainctl/), and configure the credential helper:
```